rust/arrow/src/util/bit_slice_iterator.rs

```diff
@@ -0,0 +1,108 @@
+#[cfg(all(test, target_endian = "big"))]
```

Comment:
File needs license header @vertexclique

Reply:
I think it will come from here Neville https://github.com/apache/arrow/pull/8664/files#diff-715ddfbc534281523a73117d3bf4986d69d8a375dd320bc19f83298d3ba7ebd6R1

I will rebase this branch on top of that one, waiting for it to be merged.